Mathlib/LinearAlgebra/Finsupp/Defs.lean

```diff
@@ -248,6 +248,7 @@ This is the `LinearEquiv` version of `Finsupp.finsuppProdEquiv`. -/
 @[simps]
 noncomputable def finsuppProdLEquiv {α β : Type*} (R : Type*) {M : Type*} [Semiring R]
     [AddCommMonoid M] [Module R M] : (α × β →₀ M) ≃ₗ[R] α →₀ β →₀ M :=
+  letI := Classical.decEq α
```

Comment:
This is going to result in a bad `simps` lemma; can you add `DecidableEq α` to the assumptions instead?